src/widgets/StaticCallouts/hooks.js

```diff
@@ -1,8 +1,27 @@
+import { useContext, useState, useEffect } from 'react';
+
 import { breakpoints, useWindowSize } from '@edx/paragon';
+import { AppContext } from '@edx/frontend-platform/react';
+
+import { activateExperiment, getExperimentVariation } from './experiment';
 
-export const useIsMediumScreen = () => {
+const useIsMediumScreen = () => {
   const { width } = useWindowSize();
   return width >= breakpoints.medium.minWidth && width < breakpoints.medium.maxWidth;
 };
 
-export default useIsMediumScreen;
+const useOptimizelyExperiment = () => {
+  const { authenticatedUser } = useContext(AppContext);
+  const [variation, setVariation] = useState(false);
```

Comment:
This variable name isn't serving the purpose of the way it is used.
```suggestion
  const [isExperimentEnabled, setIsExperimentEnabled] = useState(false);
```

Reply:
We can also rename it to `const [isVariationGroup, setIsVariationGroup] = useState(false);` because experiment is enabled for both control and variation groups.

Reply:
boolean value and "isVariationGroup" doesn't work together. 
My the name it sounds like the value will be a string

Reply:
The value will be what we return and what we set in the initial state. If we return a boolean it will remain a boolean. 
However, I change the approach and now we are not using useState in `StaticCallouts` component and using hooks value directly.

Reply:
I rename the state from `const [variation, setVariation] = useState(false);` to `const [show2ULobs, setShow2ULobs] = useState(false);`